foundation/foundation-archetype/src/main/resources/archetype-resources/__rootArtifactId__-tests/__rootArtifactId__-tests-docker/pom.xml

```diff
@@ -110,6 +110,7 @@
                                             <artifactId>${project.parent.artifactId}-java</artifactId>
                                             <version>${project.version}</version>
                                             <type>jar</type>
+                                            <classifier>shaded</classifier>
```

Comment:
i: Updated the shading in `*-test-java` and here to resolve issue with the copied jar not being executable.